src/control/index.ts

```diff
@@ -53,9 +82,9 @@ export class RulerControl extends mapgl.Control {
     private render = () => {
         this.getContainer().innerHTML = `
             <div class=${styles.root}>
-                <button class="${styles.button} ${
-            this.isEnabled ? styles.enabled : undefined
-        }"> ${icon}</button>
+                <button class="${styles.button} ${this.isEnabled ? styles.enabled : undefined}"> ${
```

Comment:
при таком подходе у тебя в дом добавится класс undefined

```suggestion
                <button class="${styles.button}${this.isEnabled ? ' ' + styles.enabled : ''}"> ${
```